scripts/cli.sh

```diff
@@ -22,6 +22,11 @@ else
     tty_arg=""
 fi
 
+# We want the args as a string that gets added to the python command passed to
+# bash.
+# shellcheck disable=SC2124
+ARGS_STR=$@
```

Comment:
I think we should quote `$@` here!

Reply:
[5d23891](https://github.com/nsidc/qgreenland/pull/498/commits/5d23891fabd7cdf206c97314426da36ff32e8aec)